doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc

```diff
@@ -0,0 +1,71 @@
+[id='optashift-ER-building-eclipse-proc']
+= Building and running the employee rostering starter application using Eclipse
+You can use Eclipse, including Red Hat JBoss Development Studio, to build the employee rostering starter application and run it.
+
+.Prerequisites
+. Eclipse must be installed.
+. The host must have access to the Internet (for downloading Maven packages from external repositories).
+. To run the application with the suggested configuration, Google Chrome must be installed. However, you can modify the configuration to use another web browser.
+
+.Procedure
+. Start Eclipse.
+. From the main menu, select *File* > *Import...*
+. Select the *Maven* > *Existing Maven projects* wizard.
+. For the root directory, select the root directory of the application source.
+. Click *Finish*.
+. Optionally, to avoid seeing many errors in Eclipse:
+.. In the root directory of the application source, run the `mvn clean install` command and wait for the build to finish.
+.. In the Eclipse navigation tree, right-click `employee-rostering-shared` and select *Build Path* > *Configure Build Path...*
+.. Click the *Source* tab, then *Add Folder...*
+.. Select the `employee-rostering-shared/target/generated-sources` folder and click *OK*.
+. From the main menu, select *Run* > *External Tools* > *External Tools Configurations...*. Under *Program*, create the following launch configurations:
+.. Open OptaWeb Employee Rostering in Chrome:
```

Comment:
If  OptaWeb Employee Rostering is UI element, bold.

Reply:
It is not.